src/components/UserItem.jsx

```diff
@@ -0,0 +1,21 @@
+export default function UserItem({
+  avatar,
+  first_name,
+  last_name,
```

Comment:
In javascript, we are supposed to use camel case instead of snake case.
I have seen that those two are relying on the API data for that you can rename them as you destructure them.